### PR TITLE
Map classification to descriptive metadata for cocina

### LINKED ISSUE #1094: Map cocina descriptive classification
When I call `Cocina::ToFedora::Descriptive.transform` with:
```
{
  "subject": [
    {
      "type": "classification",
      "value": "G9801.S12 2015 .Z3",
      "source": {
        "code": "lcc"
      }
    }
  ]
}
```

It should have `<classification>` nodes in the descMetadata
```
<classification authority="lcc">G9801.S12 2015 .Z3</classification>
```

https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_classification.txt

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Fixes #1094 

Blocked by adding to cocina data model.

## How was this change tested?

Unit.

## Which documentation and/or configurations were updated?

N/A

### CODE REVIEW

File: app/services/cocina/to_fedora/descriptive/classification.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Cocina
+  module ToFedora
+    class Descriptive
+      # Maps geo extension from cocina to MODS
```
Comment (jcoyne) [inline app/services/cocina/to_fedora/descriptive/classification.rb:6]: not geo

File: app/services/cocina/to_fedora/descriptive/classification.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        def initialize(xml:, classification:)
+          @xml = xml
+          @classification = classification
+        end
+
+        def write
+          return if classification.nil?
+
+          attributes = {}
```
Comment (jcoyne) [inline app/services/cocina/to_fedora/descriptive/classification.rb:24]: we've been leaning heavily on compact elsewhere, can we do that here?

for example:
```ruby
attributes = { whatever: maybe_nil }.compact
```

File: app/services/cocina/to_fedora/descriptive/classification.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+        private
+
+        attr_reader :xml, :classification
+
+        def content
+          subject.first[:value] if subject
+        end
+
+        def subject
```
Comment (jcoyne) [inline app/services/cocina/to_fedora/descriptive/classification.rb:39]: can we change this to first_subject and return:
```
          classification.first[:subject]&.first
```

that should reduce the need for several conditionals in other methods.

File: app/services/cocina/to_fedora/descriptive/classification.rb (hunk 2 above)
Comment (aaron-collier) [reply app/services/cocina/to_fedora/descriptive/classification.rb:24]: I like this. Thanks.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**